lib/improver/spotdata/write_output.py

```diff
@@ -34,7 +34,7 @@
 import iris
 from iris import FUTURE
```

Comment:
Unused import FUTURE

Reply:
Ha! yes, know the feeling... 
But this seems to do what's required, tests pass for me and I can't find any missed iris.save() or iris.load statements.